spec/features/bulk_jobs_spec.rb

```diff
@@ -31,7 +31,7 @@
   end
 
   context 'on the form for creating a new bulk job', js: true do
-    let(:bulk_job_data_path) { File.expand_path('../fixtures/crowdsourcing_bridget_1.xml', __dir__) }
+    let(:bulk_job_data_path) { File.join(file_fixture_path, 'crowdsourcing_bridget_1.xml') }
```

Comment:
Can you use the Rails-provided helper here? `file_fixture('crowdsourcing_bridget_1.xml')` ?

Reply:
Yes, in some places. I was unable to in `spec/services/modsulator_client_spec.rb` and I don't really want to go deep on this.

Reply:
@jcoyne ok, fine by me